src/rpc/server.cpp

```diff
@@ -194,8 +194,6 @@ std::string CRPCTable::help(const std::string& strCommand) const
         const CRPCCommand *pcmd = command.second;
         string strMethod = pcmd->name;
         // We already filter duplicates, but these deprecated screw up the sort order
```

Comment:
It used to have other deprecated commands, long-ago deleted. I'm not actually sure what the comment is trying to say.